spring-cloud-stream-binders/spring-cloud-stream-binder-kafka/pom.xml

```diff
@@ -63,12 +72,6 @@
 			<groupId>org.springframework.xd</groupId>
 			<artifactId>spring-xd-tuple</artifactId>
 			<version>${spring-xd.version}</version>
-			<exclusions>
```

Comment:
spring-xd-tuple still depends on spring-xd-codec. Previously, we had the same classes (and package names) in s-c-s-codec since they were also referenced in the binders. Now there is no longer a conflict. The plan is to eventually break out tuple as spring project.